tools/bazar/presentation/javascripts/components/BazarSearch.js

```diff
@@ -0,0 +1,53 @@
+// TODO better list and translatable
+var wordsToExcludeFromSearch = ['le', 'la', 'les', 'du', 'en', 'un', 'une']
+
+export default {
+  methods: {
+    searchEntries(entries, search) {
+      switch (this.params.search) {
+        case "dynamic":
+          return this.localSearch(entries, search)
+        case "true":
+          return this.distantSearch(entries, search)
+        default:
+          return entries
+      }
+    },
+    // Search throught API
+    distantSearch(entries, search) {
+      let params = { ...this.params, ...{ q: search } }
```

Comment:
pour réduire le volume de données retournées, il me semble qu'on peut mettre(à tester)
```suggestion
      let params = { ...this.params, ...{ q: search,displayfields:'id_fiche' } }
```

Reply:
non displayFields ajoute des champs (en plus de ceux déjà nécessaire, i.e id_fiche et bf_titre)
Mais là de base y'a que id_fiche et bf_titre donc ça c'est pas trop large !